include/cppflowCM/InterfaceCM.h

```diff
@@ -1,47 +1,38 @@
 #pragma once
 
+#include <string>
+
 #include <opencv2/opencv.hpp>
 #include <cppflow/Tensor.h>
 #include <cppflow/Model.h>
 #include <cppflowATRInterface/Object_Detection_Types.h>
 
-#include <string>
+#include "InterfaceCMbase.h"
 
-class mbInterfaceCM
+class mbInterfaceCM : public mbInterfaceCMbase
 {
 protected:
   Model *m_model;
   Tensor *m_inTensorPatches;
   Tensor *m_outTensorScores;
 
-  int m_patchHeight = 128;
-  int m_patchWidth = 128;
+  // int m_patchHeight = 128;
+  // int m_patchWidth = 128;
+  // int m_batchSize = 32;
+  // int m_numColors = 7;
+  // bool m_hardBatchSize = false;
+  //float m_tileMargin = 0.2;
 
-#ifdef TEST_MODE
```

Comment:
I need this test mode

Reply:
resolved, but in 'src/Object_Detection_Handler.cpp'.